packages/generator-sketch/.gitignore

```diff
@@ -1,5 +1,6 @@
 /dist
 /node_modules
+/ChangeLog
```

Comment:
what's this? is it targeting packages/generator-sketch/CHANGELOG.md ? I see new additions to that file below - do you want to remove that from git, or what's the plan?

Reply:
yep, that _was_ the plan at least 😅
but i realized this will be used to track the sketch build object changes
(was a bit overwhelming in size, so i thought it was an build artifact, that shoud be ignored)

will revert^^

Reply:
or not…
seems there is some issue with the tracking system for generating the change log
building it twice results in  pseudo changes to the sketch file, which shouldn't be tracked (so the changelog would keep growing although nothing changed really 🤷‍♂️)
@nowseemee maybe we shouldn't track this for now afterall (i'll create an issue though)

Reply:
so this /ChangeLog is useless?

Reply:
hopefully not
i took a quick look at the .sqlite DB for reference yesterday, but my eyes couldn't find a meaningful pattern why this the ids seem to increment
let's take look maybe in pair

Reply:
if it's not targeting  packages/generator-sketch/CHANGELOG.md  then its fine